main.py

```diff
@@ -1,10 +1,67 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
+def create_race(race_info: dict) -> None:
+    if not Race.objects.filter(name=race_info["name"]).exists():
+        Race.objects.create(
+            name=race_info["name"],
+            description=race_info["description"]
+        )
+
+
+def get_race_id(race_info: dict) -> int:
+    return Race.objects.get(name=race_info["name"]).id
+
+
+def create_skills(race_info: dict) -> None:
+    skills_list = race_info["skills"]
```

Comment:
could u use get method here, to remove check from 21 line ? 
f.e. `skills_list = race_info.get("skills", [])`